models/types/geometry_type.rb

```diff
@@ -0,0 +1,44 @@
+# Copyright © Mapotempo, 2021
+#
+# This file is part of Mapotempo.
+#
+# Mapotempo is free software. You can redistribute it and/or
+# modify since you respect the terms of the GNU Affero General
+# Public License as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+#
+# Mapotempo is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the Licenses for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Mapotempo. If not, see:
+# <http://www.gnu.org/licenses/agpl.html>
+#
+
+class GeometryType
+  ALL_TYPES = %i[polylines partitions].freeze
```

Comment:
We decided that we can always return points because it is not costly (and in fact front page would look weird if we do not return points at least). 

We can change this in a second version, while adjusting with front behaviour but I think it is okay to always return points.